### PR TITLE
Refactor package to include products code

### PR DESCRIPTION
**Pull request by mhidas**
This is in preparation for adding all the product-generation code into this package, so it can be separate from aodndata (which is pipeline-specific).

This repo will be renamed to **python-aodntools**.

### CODE REVIEW

**Comment by mhidas** [comment]
@lwgordonimos I think we'll also need to get rid of versioneer, and switch to the new way of managing package versions.

**Comment by mhidas** [comment]
@lwgordonimos I have rebased this onto the "bumpversion" version and tried to fix up the new config files to match the new package structure. Hope this works.

File: aodntools/ncwriter/imos_template.py [hunk 1]
```diff
@@ -3,8 +3,7 @@
 
 from netCDF4 import num2date
 
-from ncwriter import DatasetTemplate
-
+from aodntools.ncwriter import DatasetTemplate
```
Comment (lwgordonimos) [inline aodntools/ncwriter/imos_template.py]: The internal package imports are all relative except this one. Maybe should be:

```python
from .template import DatasetTemplate
```

?

Comment (mhidas) [reply aodntools/ncwriter/imos_template.py]: :+1: Done

**Comment by mhidas** [comment]
Thanks @lwgordonimos . I've fixed that relative import, and added the config.cfg file.

**Pull request merged**